decode.go

```diff
@@ -335,7 +335,8 @@ func (d *Decoder) DecodeInterface() (interface{}, error) {
 		return d.DecodeMap()
 	case codes.FixExt1, codes.FixExt2, codes.FixExt4, codes.FixExt8, codes.FixExt16,
 		codes.Ext8, codes.Ext16, codes.Ext32:
-		return d.extInterface(c)
+		d.s.UnreadByte()
+		return d.extInterface()
```

Comment:
Can you leave this unchanged? We try to not unread anything for better performance.